Sources/Clappr_iOS/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -450,16 +450,36 @@ open class AVFoundationPlayback: Playback {
         seekOnReadyIfNeeded()
 
         if let subtitles = self.subtitles {
-            trigger(.subtitleAvailable, userInfo: ["subtitles": subtitles])
+            let hasDefault = selectDefaultSubtitleIfNeeded()
+            trigger(.subtitleAvailable, userInfo: ["subtitles": subtitles, "hasDefaultFromOption": hasDefault])
         }
 
         if let audioSources = self.audioSources {
-            trigger(.audioAvailable, userInfo: ["audios": audioSources])
+            let hasDefault = selectDefaultAudioIfNeeded()
+            trigger(.audioAvailable, userInfo: ["audios": audioSources, "hasDefaultFromOption": hasDefault])
         }
 
         addTimeElapsedCallback()
     }
 
+    fileprivate func selectDefaultSubtitleIfNeeded() -> Bool {
```

Comment:
Why do we need to return a Bool value?  I think that we have more than one responsibility in this method.